doc/user/pathd.rst

```diff
@@ -368,38 +339,32 @@ The asterisk (*) marks the best, e.g. active, candidate path. Note that for segm
 retrieved via PCEP a random number based name is generated.
 
 
-.. index:: show debugging pathd
 .. clicmd:: show debugging pathd
```

Comment:
wow - is that the syntax - not "show pathd ..." ?

Reply:
I didn't write it

Reply:
good catch this comment doesn't even exist